contribute.html

```diff
@@ -56,54 +56,72 @@ <h4 class="p-heading-icon__title"><a href="https://github.com/lyubomir-popov">ly
 <div class="p-strip">
   <div class="row">
     <div class="col-6">
       <div class="p-heading-icon">
-        <div class="p-heading-icon__header">
-          <svg width="60" height="60" class="p-heading-icon__img" viewBox="0 0 12 16" version="1.1" aria-hidden="true"><title>GitHub pull requist icon</title><path fill-rule="evenodd" d="M11 11.28V5c-.03-.78-.34-1.47-.94-2.06C9.46 2.35 8.78 2.03 8 2H7V0L4 3l3 3V4h1c.27.02.48.11.69.31.21.2.3.42.31.69v6.28A1.993 1.993 0 0 0 10 15a1.993 1.993 0 0 0 1-3.72zm-1 2.92c-.66 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2zM4 3c0-1.11-.89-2-2-2a1.993 1.993 0 0 0-1 3.72v6.56A1.993 1.993 0 0 0 2 15a1.993 1.993 0 0 0 1-3.72V4.72c.59-.34 1-.98 1-1.72zm-.8 10c0 .66-.55 1.2-1.2 1.2-.65 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2zM2 4.2C1.34 4.2.8 3.65.8 3c0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2z"></path></svg>
+        <div class="p-heading-icon__header is-stacked">
+          <svg width="60" height="60" class="p-heading-icon__img" viewBox="0 0 12 16" version="1.1" aria-hidden="true">
+            <title>GitHub pull requist icon</title>
+            <path fill-rule="evenodd"
+              d="M11 11.28V5c-.03-.78-.34-1.47-.94-2.06C9.46 2.35 8.78 2.03 8 2H7V0L4 3l3 3V4h1c.27.02.48.11.69.31.21.2.3.42.31.69v6.28A1.993 1.993 0 0 0 10 15a1.993 1.993 0 0 0 1-3.72zm-1 2.92c-.66 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2zM4 3c0-1.11-.89-2-2-2a1.993 1.993 0 0 0-1 3.72v6.56A1.993 1.993 0 0 0 2 15a1.993 1.993 0 0 0 1-3.72V4.72c.59-.34 1-.98 1-1.72zm-.8 10c0 .66-.55 1.2-1.2 1.2-.65 0-1.2-.55-1.2-1.2 0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2zM2 4.2C1.34 4.2.8 3.65.8 3c0-.65.55-1.2 1.2-1.2.65 0 1.2.55 1.2 1.2 0 .65-.55 1.2-1.2 1.2z">
+            </path>
+          </svg>
           <h3 class="p-heading-icon__title">Running tests locally</h3>
         </div>
       </div>
-      <p>The simplest way to run Vanilla framework is to first <a href="https://docs.docker.com/engine/installation/" class="p-link--external">install Docker</a>, and then use the <code>./run</code> script to <a href="https://github.com/canonical-web-and-design/vanilla-framework#vanilla-local-development" class="p-link--external">build the site</a>. Before proposing a pull request, ensure that the tests pass on your local fork. To kick off the tests for your project, in the terminal <code>./run test</code>.</p>
+      <p>The simplest way to run Vanilla framework is to first <a href="https://docs.docker.com/engine/installation/" class="p-link--external">install Docker</a>, and then use the <code>./run</code> script to <a
+          href="https://github.com/canonical-web-and-design/vanilla-framework#vanilla-local-development" class="p-link--external">build the site</a>. Before proposing a pull request, ensure that the tests pass on your local fork. To kick off
+        the
+        tests for your project, in the terminal <code>./run test</code>.</p>
```

Comment:
Some weird new lines in the code here and above

Reply:
Fixed 👍